### PR TITLE
Plugin model

### PR DESCRIPTION
**Pull request by klumba12**

### CODE REVIEW

File: docs/sandbox.md [hunk 1]
```diff
@@ -2,103 +2,78 @@
...
-    $scope.onColumnOrderChanged = function(e){
-    };
+controller.$inject = ['qgrid', '$http'];
+function controller(qgrid, $http){
+	const gridModel = qgrid.model();
+	const gridService = qgrid.service(gridModel);
+	
+	gridModel.data({
+		pipe: [
+				(data, model, next) => {
```
Comment (vkorolev) [inline docs/sandbox.md:66]: Context instead of model, I guarantee :)

File: src/core/pagination/pagination.model.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import Resource from '../resource/resource';
+
+export default class PaginationModel {
+	constructor() {
+		this.resource = new Resource();
```
Comment (vkorolev) [inline src/core/pagination/pagination.model.js:5]: Don't we need a current page as well?

File: src/core/services/utility.js [hunk 3]
```diff
@@ -26,6 +28,9 @@ export {
 	isUndefined,
 	clone,
 	debounce,
+	merge,
+	yes,
```
Comment (vkorolev) [inline src/core/services/utility.js:32]: How about `truthy`|`falsy` or even `true` | `false` ?
Similar naming is used in assertion libs

File: src/core/pagination/pagination.model.js (hunk 2 above)
Comment (klumba12) [reply src/core/pagination/pagination.model.js:5]: current will be a part of scroll. maybe view, maybe of pagination, so for now let it be fake current)

File: src/core/services/utility.js (hunk 3 above)
Comment (klumba12) [reply src/core/services/utility.js:32]: true|false is system stuff don't wont to mix it, truthy vs yes can make vote in the future)

File: docs/sandbox.md (hunk 1 above)
Comment (klumba12) [reply docs/sandbox.md:66]: =) yep) it'll continously updated bro =)

**Review by vkorolev — APPROVED** [review]

**Pull request merged**